index.ts

```diff
@@ -39,6 +42,25 @@ function getConfig(arc: {
   else return {}
 }
 
+const openSearchApiFile = 'openSearchApi.js'
+
+async function executeSearchRequests(cwd: string) {
+  //Load api call file and run all api calls to cluster
+  const apiPath = join(cwd, openSearchApiFile)
+  let result
+  if (await exists(apiPath)) {
+    console.log(`Found ${openSearchApiFile} file, running api calls...`)
+    result = (await import(pathToFileURL(apiPath).toString())).default
+
+    // result should be a function that returns a promise
+    if (typeof result === 'function') {
+      result = result()
```

Comment:
Yes.

Reply:
Yes.

Reply:
Alright, done. It seems to work in sandbox mode and it should hopefully do the same for deployment.

Reply:
Just a quick note, since we're now getting the configured client from architect-functions-search, won't this plugin only work for OpenSearch now, since architect-functions-search only configures a client for OpenSearch?